sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -3,244 +3,159 @@
       <Count>1</Count>
     </item>
   </SALCommand>
   <SALCommand>
     <Subsystem>MTAOS</Subsystem>
-    <EFDB_Topic>MTAOS_command_processIntraExtraWavefrontError</EFDB_Topic>
-    <Description>Process the wavefront data collection by camera piston.</Description>
+    <EFDB_Topic>MTAOS_command_preProcess</EFDB_Topic>
+    <Description>Pre-process image for WEP. The outputs of this command are donut images that are ready for curvature wavefront sensing.</Description>
     <item>
-      <EFDB_Name>intraVisit</EFDB_Name>
-      <Description>Intra-focal image visit number.</Description>
-      <IDL_Type>int</IDL_Type>
+      <EFDB_Name>visitId</EFDB_Name>
+      <Description>Image visit id number. This is a unique identifier for each image and consists of the date of the observation plus an integer iterator, e.g. 2020103000040 for the 40th image taken in October, 30, 2020.</Description>
+      <IDL_Type>long</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>extraVisit</EFDB_Name>
-      <Description>Extra-focal image visit number.</Description>
-      <IDL_Type>int</IDL_Type>
+      <EFDB_Name>useOCPS</EFDB_Name>
+      <Description>Use the OCS-Controlled Pipeline System (OCPS) to process data remotely?</Description>
+      <IDL_Type>boolean</IDL_Type>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
     <item>
-      <EFDB_Name>intraDirectoryPath</EFDB_Name>
-      <Description>Absolute directory path to intra-focal wavefront data collection.</Description>
+      <EFDB_Name>config</EFDB_Name>
+      <Description>Yaml configuration string with additional parameters to be passed to ISR and other pre-process tasks. If options are invalid command will be rejected. If not given, use the standard configuration.</Description>
       <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
       <Units>unitless</Units>
       <Count>1</Count>
     </item>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_command_runWEP</EFDB_Topic>
+    <Description>Process wavefront data, either full focal plane mode (Comcam and LSSTCam) or wavefront sensors. If only visitId is provided it assumes wavefront sensor mode. If visitId and extraId is provided it assumes full focal plane mode. MTAOS will determine if it is ComCam or LSSTCam based on the input images. If useOCPS=True, processing is done remotely using the OCPS CSC. For LSSTCam full array mode only OCPS processing is supported. Before running this command it is advisable to run selectSources and preProcess commands on the input image(s), otherwise MTAOS will have to run those processes before running wep.</Description>
     <item>
-      <EFDB_Name>extraDirectoryPath</EFDB_Name>
-      <Description>Absolute directory path to extra-focal wavefront data collection.</Description>
-      <IDL_Type>string</IDL_Type>
-      <IDL_Size>1</IDL_Size>
+      <EFDB_Name>visitId</EFDB_Name>
```

Comment:
If the `extraId` is given, the `visitId` will be the intra-focal or extra-focal image? Can you add this in the description?

Reply:
You put in the `extraId` already. Sorry!